rplugin/python3/denite/source/output.py

```diff
@@ -17,6 +17,55 @@ def __init__(self, vim):
         # why doesn't this seem to be working?
         self.default_action = 'yank'
 
+    def highlight(self):
+        cmd = self.context['args'][0]
+        if re.fullmatch(r'hi(ghlight)?(!)?', cmd):
+            hi_list = self.vim.call('execute', cmd).splitlines()[1:]
+            for hi in hi_list:
+                self.define_highlight_for_hi(hi.split()[0])
+
+    def define_highlight_for_hi(self, hi):
+        hi_name = (
+            'highlight default link' +
+            ' deniteSource_outputHighlight_' + hi +
+            ' vimGroupName'
+        )
+        hi_xxx = (
+            'highlight default link' +
+            ' deniteSource_outputHighlight_' + hi + 'xxx' +
+            ' ' + hi
+        )
+        self.vim.command(hi_name)
+        self.vim.command(hi_xxx)
+
+    def define_syntax(self):
+        cmd = self.context['args'][0]
+        if re.fullmatch(r'hi(ghlight)?(!)?', cmd):
+            self.vim.command('syntax include syntax/vim.vim')
```

Comment:
I think it should be in `self.deine_syntax_for_hi()`.
`define_syntax()` should not be long.